docs/includes/guides/migration_3.x.adoc

```diff
@@ -0,0 +1,63 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/../..]
+
+== Java 17 Runtime
+
+Java 11 is no longer supported in Helidon 3. Java 17 or newer is required.
+
+
+== javax.* workspace to jakarta.* workspace
```

Comment:
workspace -> namespace (or package)

Reply:
This is not relevant for Helidon SE, it can be moved to MP

Reply:
moved to mp